discord_thread_summarizer_using_openai/main.bal

```diff
@@ -0,0 +1,76 @@
+import ballerina/io;
+import ballerina/time;
+import ballerina/log;
```

Comment:
```suggestion
import ballerina/io;
import ballerina/log;
import ballerina/time;
```

Reply:
Addressed in https://github.com/ballerina-guides/ai-samples/pull/69/commits/ca9ecd0b4475145df3519080000a28a26d049d7d